mon-pix/app/components/certification-joiner.js

```diff
@@ -60,17 +58,27 @@ export default class CertificationJoiner extends Component {
     this.showCongratulationsBanner = false;
   }
 
+  _isAValidSessionId(value) {
+    const isANumberRegex = new RegExp(this.sessionPattern);
+    return isANumberRegex.test(value);
+  }
+
+  @action
+  checkIsANumber(event) {
```

Comment:
j'inverserai les noms des deux nouvelles fonctions. Ici l'action c'est bien de vérifier que l'id de session est valide, donc l'action serait `checkSessionIdIsValid(event)`.
et pour la fonction avec la regex, `this._isANumber(value)`